ibm_catalog.json

```diff
@@ -0,0 +1,92 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-icd-elasticsearch",
+        "label": "Elasticsearch",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "reference_architecture",
```

Comment:
remove `reference_architecture` - its not a supported tag
add `databases`

(FYI all supported tags can be seen by running `ibmcloud catalog filter options --all`)